synapse/handlers/federation.py

```diff
@@ -86,9 +86,14 @@
     "sec",
```

Comment:
If the changes look good, please merge. I'd like to get some baseline data on `matrix.org`.